recipes/aiida-core/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "aiida-core" %}
+{% set version = "0.12.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 482e7aff408d6cc21e8ddde22efaa124d71619de53eccd37174c8e6d7e7e141e
+
+build:
+  number: 0
+  entry_points:
+    - verdi=aiida.cmdline.verdilib:run
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  skip: True  # [py > 27]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - reentry ==1.2.2
+    - python-dateutil ==2.7.2
+    - python-mimeparse ==1.6.0
+    - django ==1.7.11
+    - django-extensions ==1.5.0
+    - tzlocal ==1.5.1
+    - pytz ==2018.4
+    - PyYAML ==3.12
+    - six ==1.11.0
+    - future ==0.16.0
+    - pathlib2 ==2.3.0
+    - celery ==3.1.25
+    - billiard ==3.3.0.23
+    - amqp ==1.4.9
+    - anyjson ==0.3.3
+    - plumpy ==0.7.12
+    - portalocker ==1.1.0
+    - psutil ==5.4.5
+    - meld3 ==1.0.2
+    - numpy ==1.14.3
+    - SQLAlchemy ==1.0.19
+    - SQLAlchemy-Utils ==0.33.0
+    - alembic ==0.9.9
+    - ujson ==1.35
+    - enum34 ==1.1.6
+    - voluptuous ==0.11.1
+    - aldjemy ==0.8.0
+    - passlib ==1.7.1
+    - validate_email ==1.3
+    - click ==6.7
+    - click-plugins ==1.0.3
+    - click-spinner ==0.1.7
+    - tabulate ==0.8.2
+    - ete3 ==3.1.1
+    - uritools ==2.1.0
+    - psycopg2 ==2.7.4
+    - paramiko ==2.4.2
+    - ecdsa ==0.13
+    - ipython <6.0
+
+test:
+  imports:
+    - aiida
+    - aiida.cmdline
+    - aiida.common
+    - aiida.control
+    - aiida.restapi
+    - aiida.scheduler
+    - aiida.transport
+    - aiida.utils
+
+about:
+  home: http://aiida.net
+  license: MIT
+  license_family: MIT
+  # Remove once this PR has been merged
+  # https://github.com/aiidateam/aiida_core/pull/2339
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
+  summary: AiiDA, an automated interactive infrastructure and database for computational science
+  doc_url: https://aiida-core.readthedocs.io
+  dev_url: https://github.com/aiidateam/aiida_core
+
+extra:
+  recipe-maintainers:
+    - ltalirz
+    - astamminger
```

Comment:
@astamminger are you okay with being a maintainer?

Reply:
@CJ-Wright yes i am

Reply:
@CJ-Wright build passing!